src/test/java/FlightsReporterTests.groovy

```diff
@@ -0,0 +1,197 @@
+import org.junit.Before
+import org.junit.Test
+import org.junit.Assert
+import org.junit.jupiter.api.BeforeAll
+
+import java.nio.channels.FileLock
+import java.time.LocalDateTime
+import java.time.LocalTime;
+//import FlightsReporter
+
+import static  org.junit.jupiter.api.Assertions.*;
+
+class FlightsReporterTests {
+    List<Flight> flights = []
+
+    @Test
+    void testFlightLengthInMinutes() {
+        Flight fl1 = new Flight(id: "SOFLON11", from: FlightsReporter.Airports.SOFIA, destination: FlightsReporter.Airports.LONDON,
+            departure: LocalDateTime.of(2021, 8, 3, 10, 30),
+                arrival: LocalDateTime.of(2021, 8, 3, 13, 15),
+                passengers: 100, capacity: FlightsReporter.CAPACITIES[3], runningLate: false)
+
+        assertEquals(FlightsReporter.flightLengthInMinutes(fl1), 165)
+
+    }
+
+    @Test
+    void testGetAllFlightsTo() {
+        FlightsReporter.printFlights(flights)
+        assert FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.AMSTERDAM)
+                == [flights[0], flights[1]]
+
+        assertEquals(FlightsReporter.getAllFlightsTo(flights, FlightsReporter.Airports.SOFIA),
+                Arrays.asList(flights[8], flights[9]))
```

Comment:
You depend on indexes to find a flight and assert it. This makes you bound to the order you create your test data... 8 doesn't really mean much. or 9 . Magic numbers

Try to think of a better way express that in code.